crates/biome_cli/src/commands/format.rs

```diff
@@ -39,12 +42,27 @@ pub(crate) fn format(
         write,
         json_formatter,
     } = payload;
+    let loaded_configuration = load_configuration(&mut session, &cli_options)?.with_file_path();
+
+    loaded_configuration.check_for_errors(session.app.console, cli_options.verbose)?;
+
     let LoadedConfiguration {
         mut configuration,
         directory_path: configuration_path,
         ..
-    } = load_configuration(&mut session, &cli_options)?
-        .or_diagnostic(session.app.console, cli_options.verbose)?;
+    } = loaded_configuration;
+    if formatter_configuration
+        .as_ref()
+        .is_some_and(|f| f.indent_size.is_some())
+    {
+        let console = &mut session.app.console;
+        let diagnostic = DeprecatedArgument::new(markup! {
+            "The argument "<Emphasis>"--indent-size"</Emphasis>"is deprecated, it will be removed in the next major release. Use "<Emphasis>"--indent-width"</Emphasis>" instead."
```

Comment:
A space is Missing:
```suggestion
            "The argument "<Emphasis>"--indent-size"</Emphasis>" is deprecated, it will be removed in the next major release. Use "<Emphasis>"--indent-width"</Emphasis>" instead."
```